src/components/SiteHeader/SiteHeader.tsx

```diff
@@ -1,15 +1,17 @@
 import 'styles/base.css'
 
+import classNames from 'classnames'
 import _ from 'lodash'
 import React from 'react'
 import Headroom from 'react-headroom'
+import { Flex, Heading } from 'rebass/styled-components'
 
+import HeaderLink from 'components/SiteHeader/HeaderLink'
 import GatsbyLocation from 'types/GatsbyLocation'
 
-import HeaderLink from './HeaderLink'
-
-interface Props {
-  location: GatsbyLocation
+export enum HeaderTheme {
+  DARK = 'DARK',
+  LIGHT = 'LIGHT',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 2. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)